scripts/hudson/narayana.bat

```diff
@@ -0,0 +1,75 @@
+if not defined WORKSPACE (call:fail_build & exit -1)
+
+set NOPAUSE=true
+@echo off
+
+call:comment_on_pull "Started testing this pull request on Windows: %BUILD_URL%"
+
+java -version 2>tmp.txt
+findstr version tmp.txt > version.txt
+FOR /F delims^=^"^ tokens^=2 %%i in (version.txt) DO @set JAVA_VERSION=%%i
+if %JAVA_VERSION% == 9 set MAVEN_OPTS="--add-modules=java.corba,java.xml.bind,java.xml.ws"
+
+IF NOT [%NOTMAIN%] == [] (call:build_spi_pr
+call build.bat clean install %* || (call:comment_on_pull "Tests failed on Windows - Narayana Failed %BUILD_URL%" & exit -1)
+)
+
+IF NOT [%NOTBT%] == [] (echo "set WILDFLY_MASTER_VERSION=%WILDFLY_MASTER_VERSION% set JBOSSAS_IP_ADDR=%JBOSSAS_IP_ADDR%"
```

Comment:
NOTBT is referring to not being BlackTie but BlackTie no longer exists, ie it is confusing.

Reply:
I also believe that `NOT [%NOTBT%] == []` is saying if it is BlackTie

Reply:
okay, I will remove this